components/map/MapZoneBottomSheet.tsx

```diff
@@ -112,139 +80,60 @@ const MapZoneBottomSheet = forwardRef<BottomSheet, Props>((props, ref) => {
-                    {t('Common.cancel')}
-                  </Button>
-                )}
-              </FlexRow>
-            </View>
-            <View className="flex-1">
-              {isFullHeightEnabled && (
-                <Pressable onTouchStart={handleInputBlur}>
-                  <View className="h-full">
-                    <View className="flex-1 pt-5">
-                      <PortalHost name="mapAutocompleteOptions" />
-                    </View>
-                  </View>
-                </Pressable>
-              )}
+          {isZoomedOut ? (
+            <View className="flex-col items-center">
+              <Typography>{t('zoomIn')}</Typography>
             </View>
-          </View>
-          {!isFullHeightEnabled &&
-            (selectedZone ? (
-              <>
-                <Panel className="g-4">
-                  <FlexRow>
-                    <Typography>{selectedZone.name}</Typography>
-                    <ZoneBadge label={selectedZone.udrId} />
-                  </FlexRow>
-                  <Divider />
-                  <FlexRow>
-                    <Typography variant="default-bold">
-                      {formatPricePerHour(selectedZone.price)}
-                    </Typography>
-                    <Link
-                      asChild
-                      href={{
-                        pathname: '/zone-details',
-                        params: {
-                          udrId: selectedZone.udrId,
-                        } satisfies ZoneDetailsParamas,
-                      }}
-                    >
-                      <PressableStyled>
-                        <View className="flex-row">
-                          <Typography variant="default-bold">
-                            {t('ZoneBottomSheet.showDetails')}
-                          </Typography>
-                          <Icon name="expand-more" />
-                        </View>
-                      </PressableStyled>
-                    </Link>
-                  </FlexRow>
-                </Panel>
-
-                <Link
-                  asChild
-                  href="/purchase"
-                  onPress={() => onPurchaseStoreUpdate({ udr: selectedZone })}
-                >
-                  <Button variant="primary">{t('Navigation.continue')}</Button>
-                </Link>
-              </>
-            ) : (
-              <Panel className="bg-warning-light g-2">
-                <Typography>{t('ZoneBottomSheet.noZoneSelected')}</Typography>
-              </Panel>
-            ))}
+          ) : (
+            <>
+              <MapZoneBottomSheetSearch
+                {...{
+                  ref: inputRef,
+                  handleInputBlur,
+                  isFullHeight,
+                  setIsFullHeight,
+                  selectedZone,
+                  setFlyToCenter,
+                  bottomSheetRef: localRef.current,
+                }}
+              />
```

Comment:
I would prefer standard props passing, or potentially, create separate const to be spread here (meybe without refs?).